code/modules/asset_cache/assets/tgui.dm

```diff
@@ -3,6 +3,7 @@
 	assets = list(
 		"tgui.bundle.js" = file("tgui/public/tgui.bundle.js"),
 		"tgui.bundle.css" = file("tgui/public/tgui.bundle.css"),
+		"tgui.acs.css" = file("html/acs.css"),
```

Comment:
This is also really stupid and horrible and really bad

Reply:
ghetto solution ftw